pdns/pdns_recursor.cc

```diff
@@ -1729,7 +1743,7 @@ static string* doProcessUDPQuestion(const std::string& question, const ComboAddr
     */
 #endif
 
-    if(needECS || needXPF || (t_pdl && t_pdl->d_gettag)) {
+    if(needECS || needXPF || (t_pdl && (t_pdl->d_gettag || t_pdl->d_gettag_ffi))) {
```

Comment:
on [line 1555](https://github.com/PowerDNS/pdns/pull/6344/files#diff-387fdd23379a8cda6173499eba19a125R1555) we do the gettag in a different order. If for some reason both `gettag` and `gettag_ffi` are defined, this might lead to inconsistent behaviour.

Reply:
We are just looking to see if at least one of them is defined here, so I don't think it matters? The actual invocation is done on lines 1568 and 1762, and we do consistently look for the FFI version first there.